src/index.js

```diff
@@ -1250,8 +1250,18 @@ module.exports = {
    * @param {buffer} privateKey The sender's plaintext private key.
    * @return {string} Signed and serialized raw transaction.
    */
-  signRawTransaction: function (packaged, privateKey) {
-    var rawTransaction = new EthTx(packaged);
+  signRawTransaction: function (packaged, privateKeyOrSigner, callback) {
+    try {
+      var transaction = new EthTx(packaged);
+      return (typeof privateKeyOrSigner === "function")
+        ? privateKeyOrSigner(transaction, callback)
+        : this.signTransactionWithKey(transaction, privateKeyOrSigner, callback);
+    } catch (error) {
+      return callback(error, undefined);
```

Comment:
The problem with `(error || result)` is that while this library uses `(errror || result)`, the callback will be passed through to the user-provided signer function.  Since `(error, result)` is the more common signature throughout the JavaScript ecosystem it seems like in this case we should follow that instead of forcing all users of this library to conform to the `(error || result)` style that this library uses.